jsmime.js

```diff
@@ -1,7 +1,5 @@
-define(function(require) {
```

Comment:
I was to `babel` & `browserify`to do the code prepossessing,  and use require is the easiest way to do that, if you think use ES6 module is necessary,  we could use it later.